internal/venice-common/src/main/java/com/linkedin/venice/pushstatushelper/PushStatusStoreReader.java

```diff
@@ -53,6 +53,24 @@ public PushStatusStoreReader(
     this.heartbeatExpirationTimeInSeconds = heartbeatExpirationTimeInSeconds;
   }
 
+  public Map<CharSequence, Integer> getVersionStatus(String storeName, int version) {
+    AvroSpecificStoreClient<PushStatusKey, PushStatusValue> client = getVeniceClient(storeName);
+    PushStatusKey pushStatusKey = PushStatusStoreUtils.getPushKey(version);
+    try {
+      PushStatusValue pushStatusValue = client.get(pushStatusKey).get();
```

Comment:
Nit,
Shall we specify a timeout?
Waiting forever is not a good option for most of the scenarios.

Reply:
Yeah, I added a 1-min timeout for all queries; hopefully that's enough.